pc-app/src/js/login.js

```diff
@@ -0,0 +1,128 @@
+/*global ipcRenderer*/
+
+var urlParams = new URLSearchParams(window.location.search);
```

Comment:
These variables use `var` keyword. For stylistic consistency, they should use `let` or `const` to match other files.

Reply:
Thanks for noticing. I will change it with the next commit.